runtime/master/src/main/java/org/apache/nemo/runtime/master/RuntimeMaster.java

```diff
@@ -115,6 +110,8 @@
   private final Server metricServer;
   private final MetricStore metricStore;
 
+  private static LambdaMaster lambdaMaster;
```

Comment:
why static? private final

Reply:
`private final` will require us to initialize the variable in RuntimeMaster constructor.
I was hoping to keep changes to RuntimeMaster minimal.
I'm not sure whether moving lambdaMaster initialization to constructor is a good idea.
What do you think? Is it a good idea to move lambdaMaster initialization to constructor?

Reply:
RuntimeMaster is singleton, so you don't have to worry about it